app.py

```diff
@@ -88,33 +88,58 @@ def return_code(self):
         return self._return_code
 
 
-@app.route('/execute', methods=['POST'])
-def execute():
+def _execute(request_as_string: str):
```

Comment:
```suggestion
def _execute(command: str, *arg: str):
```

Reply:
`conn.execute_command()` receives all its arguments as `*args` including the command itself, so no point in splitting the given full command as command+args.

Reply:
I agree with both of you so propose a middleground:
```python
def _execute(command: str, *arg: str):
```

- `command` is the term we use all over the place
- The `as_string` suffix is redundant
- No need for separate command and args